mmtrack/core/anchor/sot_anchor_generator.py

```diff
@@ -113,18 +114,24 @@ def single_level_grid_anchors(self,
         shift_y = torch.arange(0, feat_h, device=device) * stride[1]
 
         shift_xx, shift_yy = self._meshgrid(shift_x, shift_y)
-        shifts = torch.stack([
-            shift_xx, shift_yy,
-            torch.zeros_like(shift_xx),
-            torch.zeros_like(shift_yy)
-        ],
-                             dim=-1)
+        shifts = torch.stack([shift_xx, shift_yy, shift_xx, shift_yy], dim=-1)
         shifts = shifts.type_as(base_anchors)
 
+        # TODO
+        # first feat_w elements correspond to the first row of shifts
+        # add A anchors (1, A, 4) to K shifts (K, 1, 4) to get
+        # shifted anchors (K, A, 4), reshape to (K*A, 4)
+        # all_anchors = base_anchors[None, :, :] + shifts[:, None, :]
+        # first A rows correspond to A anchors of (0, 0) in feature map,
+        # then (0, 1), (0, 2), ...
         all_anchors = base_anchors[:, None, :] + shifts[None, :, :]
         all_anchors = all_anchors.view(-1, 4)
 
+        # transform the coordinate origin from the top left corner to the
```

Comment:
Please capitalize the first letter. Please also fix the problem in other comments。